packages/core/src/components/button/buttonGroup.tsx

```diff
@@ -38,8 +37,7 @@ export interface IButtonGroupProps extends IProps, React.HTMLProps<HTMLDivElemen
 
 // this component is simple enough that tests would be purely tautological.
 /* istanbul ignore next */
-@PureRender
-export class ButtonGroup extends React.Component<IButtonGroupProps, {}> {
+export class ButtonGroup extends React.PureComponent<IButtonGroupProps, {}> {
```

Comment:
we should be able to drop these empty `{}` state types, as the React types now provide a default: `interface Component<P = {}, S = {}>`.

though it's fine to not do this now, or ever.